File: beginner/c/1149.c
#include <stdio.h>
int main(){

    int n = 1, a, soma = 0, i;

    scanf("%d %d", &a, &n);

    while (n <= 0)
        scanf("%d", &n);

    for(i = 1; i <= n; i++){
        soma += a;
        a++;
    }

    printf("%d\n", soma);

    return 0;
}

File: beginner/javascript/1149.js
var input = require('fs').readFileSync('stdin', 'utf8');
var valores = input.split(' ');

var soma = 0, i = 0;

var A = parseInt(valores[0]);
var N = parseInt(valores[1]);

while (N <= 0) {
    N = parseInt(valores[++i]);
}

for (var c = 0; c < N; c++) {
    soma = soma + A + c;
}

console.log(soma);
